web/src/app/alerts/AlertsList.js

```diff
@@ -92,9 +93,10 @@ function getStatusFilter(s) {
 export default function AlertsList(props) {
   const classes = useStyles()
   const width = useWidth()
-  const isMobileScreenSize = isWidthDown('md', width)
+  const isMobileScreenSize = isWidthDown('xs', width)
```

Comment:
We're using `'sm'` in `App.js` to determine whether we're in a mobile size or not, and `'md'` in most other places. This should probably be `'sm'` to best reflect `App.js`

Reply:
Is this a current bug in production?

Reply:
If this is only being used for the transition on the create fab, I think it'd make more sense to just rename it to something else, since it's only being used when the snackbar has the potential to overlap with its intended placement, rather than if it's mobile or not

Reply:
I'm fine with what we have for the moment, the name just threw me off which is why I commented that we use sm/md normally. Probably how the bug got introduced in the first place if I had to guess. Maybe?:
```suggestion
  // used to transition fab above snackbar when snackbar width overlaps fab placement
  const isXS = isWidthDown('xs', width)
```

Reply:
That would be really convenient. I've been wanting to make an issue around our general layout and making our own, usable breakpoints, but don't have a concrete design yet. Def need to remember this for that!